cli/tests/cli-reference@.md.snap

```diff
@@ -1097,7 +1097,7 @@ Spans of revisions that are not included in the graph per `--revisions` are rend
 
   Possible values: `true`, `false`
 
-* `-l`, `--limit <LIMIT>` — Limit number of revisions to show
+* `-n`, `--limit <LIMIT>` — Limit number of revisions to show
```

Comment:
I prefer if we also update the tests that use `-l` now rather than leaving it for when we actually remove `-l` in 6 months or whenever

Reply:
Yeah, I figured out which tests I was missing